src/components/admin/ImageController.jsx

```diff
@@ -355,6 +361,8 @@ export default class ImageUploader extends BaseComponent {
       },
     };
 
+    let extensionErrorMsg = `To change the image's extension to . ${this.state.attemptedExt},`;
```

Comment:
Isn't there now a space between the `.` and the extension? So it'll render like `. png` `. jpeg` instead of `.png` `.jpeg`?